test/test_other.py

```diff
@@ -9736,7 +9736,31 @@ def test(dump_file):
     test('foo.wasm.dump')
     test('bar.wasm.dump')
 
-  def test_emsymbolizer(self):
+  # Runs llvm-objdump to get the address of the first occurrence of the
+  # specified line within the given function. llvm-objdump's output format
+  # example is as follows:
+  # ...
+  # 00000004 <foo>:
+  #        ...
+  #        6: 41 00         i32.const       0
+  #        ...
+  # The addresses here are the offsets to the start of the file. Returns
+  # the address string in hexadecimal.
```

Comment:
Maybe use docstring for this?   (tripple quotes multi-line string inside the func?)

Reply:
Done